src/style.css

```diff
@@ -1 +1,89 @@
 /* Styles go here */
+body {
+  display: flex;
+  flex-direction: column;
+  margin: 0;
+
+}
+
+ul {
+  padding-inline-start: 0;
+  margin-block-start: 0;
+  margin-block-end: 0;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  background-color: #fff;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.nav {
+  display: flex;
+}
+
+.nav__list {
+  display: flex;
+  list-style: none;
+}
+
+.nav__item {
+  display: flex;
+  margin-left: 20px;
+}
+
+.nav__item:first-child {
+  margin-left: 0;
+}
+
+.nav__link {
+  position: relative;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  height: 60px;
+  font-size: 12px;
+  text-transform: uppercase;
+  text-decoration: none #fff;
+  font-family: "Roboto", sans-serif;
+  font-weight: 500;
+  color: #000;
+}
+
+.logo {
+  display: flex;
+}
+
+.moyo-logo {
+  width: 40px;
+  height: 40px;
+  object-fit: contain;
+  margin: auto 0;
+}
+
+.divider {
+  flex-grow: 1;
+}
+
+.nav__link:hover {
+  color: #00acdc;
+}
+
+.is-active {
+  color: #00acdc;
+}
+
+.is-active::after {
+  position: absolute;
+  background-color: #00acdc;
```

Comment:
Move all repeatable colors to the CSS variables.

[More info.](https://developer.mozilla.org/en-US/docs/Web/CSS/Using_CSS_custom_properties)